Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -138,14 +142,23 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
                 }
             });
 
-            this.set('version', new Y.eZ.Version());
-            this.set('languageCode',e.selectedLanguageCode);
-            view.set('languageCode', e.selectedLanguageCode);
+            Y.Object.each(fields, function (field) {
```

Comment:
you are duplicating a part of `_initModels`. Also I've just realized you should update the content as well. So you should probably split somehow split `_initModels` and reuse a part of it here.